lib/build/bundler.js

```diff
@@ -170,9 +170,17 @@ exports.Bundler = class {
         //Order the bundles so that the bundle containing the config is processed last.
         let configTargetBundleIndex = this.bundles.findIndex(x => x.config.name == this.loaderOptions.configTarget);
         this.bundles.splice(this.bundles.length, 0, this.bundles.splice(configTargetBundleIndex, 1)[0]);
-        return Promise.all(this.bundles.map(x => x.write(this.project.build.targets[0])));
+        if (postBuild && typeof postBuild == 'function') {
+          //If we want to take a different action other than writing to a file, this callback is returned
+          return postBuild();
+        }
+        return this.write();
```

Comment:
@EisenbergEffect, so that function is already inside of the `doTransform` promise. I could just remove those lines like you mentioned, and then have the `index.js` modified like so:
``` javascript
exports.dest = function() {
  return bundler.build()
    .then(() => bundler.write());
}
```
I like that. I was hesitant to make a change like that initially in case people were using those functions (which I don't believe they should be). But since this is still alpha, we can get away with making that change I guess.